dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,272 @@
+      .addTemplate('Volume', 'gluster_volume_up{job = "$OCS_Cluster"}', 'volume')
+      .addRow(
+        (g.row('Volume Summary') +
+         {
+           showTitle: false,
+           panels: self._panels,
+         })
+         .addPanel(
+          g.panel('Status') +
+          statPanelArgs + 
+          g.statPanel('gluster_volume_up{volume = "$Volume"}',)+
+          {
+            valueMaps: [
+              {
+                "op": "=",
+                "text": "DOWN",
+                "value": "0"
+              },
+              {
+                "value": "1",
+                "op": "=",
+                "text": "UP"
+              }
+            ],
+            thresholds: "0.9,1.1",
+            "colors": [
+              "#cc0000",
+              "#92d400",
+              "#FFFFFF"
+            ],
+            colorValue: true,
+          }
+         )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_used_bytes_total:sum{job = "$OCS_Cluster",volume = "$Volume"} / gluster:volume_capacity_total_bytes:sum {job = "$OCS_Cluster",volume = "$Volume"}')
+        )
+        .addPanel(
+          g.panel('Available Capacity') +
+          statPanelArgs +
+          g.statPanel('gluster:volume_capacity_total_bytes:sum {job = "$OCS_Cluster",volume = "$Volume"} - gluster:volume_capacity_used_bytes_total:sum{job = "$OCS_Cluster",volume = "$Volume"}', 'decbytes')
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          statPanelArgs +
+          g.statPanel('')
+        )
+        .addPanel(
+          g.panel('Rebalance-Files') +
+          statPanelArgs +
+          g.statPanel('')
+        )
+        .addPanel(
+          g.panel('Rebalance-Size') +
+          statPanelArgs +
+          g.statPanel('')
+        )
+        .addPanel(
+          g.panel('Rebalance-Skipped') +
+          statPanelArgs +
+          g.statPanel('')
+        )
+        .addPanel(
+          g.panel('Rebalance-Failed') +
+          statPanelArgs +
+          g.statPanel('')
+        )
+        .addPanel(
+          g.panel('Capacity Utilization') +
+          graphPanelArgs +
+          g.queryPanel('gluster:volume_capacity_used_bytes_total:sum{job = "$OCS_Cluster",volume = "$Volume"}', '{{$Volume}}') +
+          g.stack + 
+          { yaxes: g.yaxes('decbytes') },
+        )
+        .addPanel(
+          g.panel('IOPS') +
+          graphPanelArgs +
+          g.queryPanel('sum(namespace_pod_name_container_name:container_cpu_usage_seconds_total:sum_rate) by (namespace)', '{{namespace}}') +
+          g.stack
+        )
+      )
+      .addRow(
+        g.row('LVM Summary')
+        .addPanel(
+          g.panel('LVM Thin Pool Metadata %') +
+          g.queryPanel('100 * gluster:volume_capacity_used_bytes_total:sum{volume = "$Volume"} / gluster:volume_capacity_available_bytes_total:sum{volume = "$Volume"}', '{{namespace}}') +
```

Comment:
We have Thinpool metrics available.

Reply:
It present in gluster-prometheus, but on runtime they are not showing up. Will add as soon as they are available